packages/error-reporting/package.json

```diff
@@ -0,0 +1,56 @@
+{
+  "name": "@google-cloud/error-reporting",
+  "description": "Stackdriver Error Reporting Client Library for Node.js",
+  "main": "index.js",
+  "repository": "GoogleCloudPlatform/google-cloud-node",
+  "scripts": {
+    "test": "nyc --exclude=\"fuzzer.js\" $(npm bin)/mocha ./test/unit/*.js",
+    "system-test": "nyc --exclude=\"error-message.js\" $(npm bin)/mocha ./test/system-test/*.js",
```

Comment:
I've run this using the system tests service account, and got all but one failure:

```

  1) Behvaiour acceptance testing Request/Response lifecycle mocking Receiving non-retryable errors Should fail:
     Error: Timeout of 5000ms exceeded. For async tests and hooks, ensure "done()" is called; if returning a Promise, ensure it resolves.


  2) Behvaiour acceptance testing Request/Response lifecycle mocking Receiving retryable errors Should retry:
     Error: Timeout of 25000ms exceeded. For async tests and hooks, ensure "done()" is called; if returning a Promise, ensure it resolves.


  3) Behvaiour acceptance testing Request/Response lifecycle mocking Using an API key Should provide the key as a query string on outgoing requests:
     Error: config.key must be a string
      at FakeConfiguration.Configuration._gatherLocalConfiguration (src\configuration.js:186:1927)
      at new Configuration (src\configuration.js:144:224)
      at FakeConfiguration (test\fixtures\configuration.js:15:1470)
      at Context.<anonymous> (test\system-test\testAuthClient.js:17:2500)

  4) Behvaiour acceptance testing Request/Response lifecycle mocking Callback-less invocation Should still execute the request:
     Error: Timeout of 2000ms exceeded. For async tests and hooks, ensure "done()" is called; if returning a Promise, ensure it resolves.


  5) Behvaiour acceptance testing System-live integration testing Client creation Using only project id As a runtime argument Should not throw on initialization:
     Error: Timeout of 10000ms exceeded. For async tests and hooks, ensure "done()" is called; if returning a Promise, ensure it resolves.


  6) Behvaiour acceptance testing System-live integration testing Client creation Using only project id As an env variable Should not throw on initialization:
     Error: Timeout of 10000ms exceeded. For async tests and hooks, ensure "done()" is called; if returning a Promise, ensure it resolves.


  7) Behvaiour acceptance testing System-live integration testing Client creation Using only project number As a runtime argument Should not throw on initialization:
     Error: Timeout of 10000ms exceeded. For async tests and hooks, ensure "done()" is called; if returning a Promise, ensure it resolves.


  8) Behvaiour acceptance testing System-live integration testing Client creation Using only project number As an env variable Should not throw on initialization:
     Error: Timeout of 10000ms exceeded. For async tests and hooks, ensure "done()" is called; if returning a Promise, ensure it resolves.


  9) Behvaiour acceptance testing System-live integration testing Error behvaiour An invalid env configuration Should callback with an error:
     Error: Timeout of 2000ms exceeded. For async tests and hooks, ensure "done()" is called; if returning a Promise, ensure it resolves.


  10) Behvaiour acceptance testing System-live integration testing Success behaviour Given a valid project id Should succeed in its request:
     Error: Timeout of 2000ms exceeded. For async tests and hooks, ensure "done()" is called; if returning a Promise, ensure it resolves.


  11) Behvaiour acceptance testing System-live integration testing Success behaviour Given a valid project number Should succeed in its request:
     Error: Timeout of 2000ms exceeded. For async tests and hooks, ensure "done()" is called; if returning a Promise, ensure it resolves.
```

Will our CI suites need to do anything to make these tests work?